pkg/controller/change/device/controller_test.go

```diff
@@ -241,9 +241,12 @@ func newChange(device device.ID) *devicechange.Change {
 		DeviceID: device,
 		Values: []*devicechange.Value{
 			{
-				Path:  "foo",
-				Value: []byte("Hello world!"),
-				Type:  devicechange.ValueType_STRING,
+				Path: "foo",
+				Value: &devicechange.TypedValue{
+					Bytes:    []byte("Hello world!"),
+					Type:     devicechange.ValueType_STRING,
+					TypeOpts: nil,
```

Comment:
in some classes you add this nil while in other you leave it empty. Is there a difference ? I believe that leaving it empty is equivalent to putting it to nil

Reply:
it just came up when i did "fill all fields" in the IDE - should be left out